### ISSUE TITLE
Can't tell slider position of disabled slider in high contrast

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#9128]
The unfilled track should be hollow, like in the active slider

**Comment by atneik** [comment]
Fixed in #9126
